reactstrap/build.boot

```diff
@@ -0,0 +1,30 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
```

Comment:
I think this library should depend on `cljsjs/react-with-addons` and `cljsjs/react-dom`.